apps/web/src/app/(private)/projects/[projectId]/page.tsx

```diff
@@ -55,3 +59,16 @@ export default async function ProjectPage({ params }: ProjectPageParams) {
 
   return redirect(url)
 }
+
+function getLastSeenDataFromCookie(projectId: string) {
+  const cookieStore = cookies()
+  const data = cookieStore.get(lastSeenCommitCookieName(Number(projectId)))
+  if (!data?.value) return {}
+
+  try {
+    const { commitUuid, documentUuid } = JSON.parse(data.value)
+    return { commitUuid, documentUuid }
+  } catch (e) {
+    return data.value as string
```

Comment:
so it's backwards compatible with the old cookies

Reply:
Return `{ lastSeenCommitUuid:  data.value as string, lastSeenDocumentUuid: undefined }` this way you can destructure safely later